docs/notebooks/06_differential_expression.ipynb

```diff
@@ -0,0 +1,1306 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.    path_to_R = "/usr/local/bioinf/R/R-4.2.3/bin/"
</pre><p>When running the entire notebook in a conda env / using a kernel from a conda en with R installed, this should not be necessary. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
R was installed in jupyter notebook and it was interfering

Reply:
Yhea, we just added it because otherwise, R from `/usr/local/bioinf/conda/bin` was used. Don't know who installed R there, though...